demo/src/pages/childWindow/ChildWindow.tsx

```diff
@@ -0,0 +1,70 @@
+import { useChildWindow } from "openfin-react-hooks";
+import { WindowOption } from "openfin/_v2/api/window/windowOption";
+import React from "react";
+
+import styles from "./ChildWindow.module.css";
+
+const codeExample = `import {useBounds} from "openfin-react-hooks";
+
+const Component = () => {
+    const bounds = useBounds();
+
+    return (
+        <div>
+            <div>Bounds: {JSON.stringify(bounds)}</div>
+        </div>
+    );
+}
+`;
+
+const WINDOW_NAME = "Child Window";
+
+const ChildWindow: React.FC = () => {
+    const windowOptions: WindowOption = {
+        autoShow: true,
+        defaultHeight: 200,
+        defaultWidth: 500,
+        maxWidth: 400,
+        name: WINDOW_NAME,
+        url: process.env.REACT_APP_CHILD_WINDOW_HTML,
+        waitForPageLoad: true,
+    };
+
+    const childWindow = useChildWindow({
+        cssUrl: process.env.REACT_APP_CHILD_WINDOW_CSS,
+        name: WINDOW_NAME,
+        parentDocument: document,
+        shouldClosePreviousOnLaunch: true,
+        windowOptions,
+    });
+
+    return (
+        <div className={styles.container}>
+            <h1 className={styles.title}>useChildWindow</h1>
+            <div className={styles.description}>
+                This hook returns allows creating and managing child windows.
```

Comment:
Would something like `This hook allows you to create and manage a single child window` be better wording here?